package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "clr-addons",
-  "version": "5.0.1",
+  "version": "5.0.1-alpha-1",
```

Comment:
5.0.2 please
we ditched the concept of alphas and betas, as the project is too small for that